app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if subject.errors.any? %>
```

Comment:
There are some bits in here that I've replicated from my other PR. WIP etc etc....